modules/GT_Enemy.php

```diff
@@ -83,12 +83,10 @@ function applyPenalty()
       ['player_name' => $this->player['player_name'], 'type' => $this->card->getType()]
     );
 
-    $game = $this->game;
-
     if ($this->card instanceof HazardCard) {
-      return $this->card->applyPenalty($game, $this->player);
+      return $this->card->applyPenalty($ths->game, $this->player);
```

Comment:
typo

Reply:
and I just realized that this typo was removed in a follow-up commit

Reply:
ha - nice. thanks for looking it over